deploy/examples/create-external-cluster-resources-tests.py

```diff
@@ -84,8 +84,9 @@ def test_method_create_cephCSIKeyring_cephFSProvisioner(self):
     def test_non_zero_return_and_error(self):
         self.rjObj.cluster.return_val = 1
         self.rjObj.cluster.err_message = "Dummy Error"
+        self.rjObj.rgw_pool_prefix = "default"
```

Comment:
set this in the upgrade test test_upgrade_user_permissions

Reply:
added